src/hash/hash.ts

```diff
@@ -1,8 +1,5 @@
-import {
-  verifySignature,
-  SchematisedDocument
-} from "@govtechsg/open-attestation";
+import { verifySignature, WrappedDocument } from "@govtechsg/open-attestation";
```

Comment:
Seemed like this is no longer needed in this folder. Might want to put it together with the `openAttestationHash` function.

Reply:
On that note, perhaps we want to think about how to have these verifiers provide utility functions as well. For instance the identity verifiers can provide a function that returns the resolved identities. Can create an issue to track it instead of doing it in this PR.

Reply:
removed and added issue